### PR TITLE
Fixes small text issue

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**
image

:cl:
bugfix: Apostrophes no longer break tickets.
/:cl:

### CODE REVIEW

**Comment by Amelia0010** [comment]
SPEED BLOODY MERGE WHEEEEEEEEEEEE

**Comment by Partheo** [comment]
What the fuck

Someone wake me up

This isn't real

**Comment by ActualRory** [comment]
https://www.youtube.com/watch?v=OBwS66EBUcY&ab_channel=Kobis11

**Comment by Ratvar** [comment]
russian servers fixed it three years ago

**Comment by AsV9** [comment]
The fix is such a hack though but whatever I do not care.

MERGE THIS.

**Comment by Partheo** [comment]
31676544-f8d06536-b335-11e7-9982-c1c135634ae1

**Comment by ShadowDeath6** [comment]
call the proc fix_apostrophes to retain some semblance of professionalism within admin code

**Comment by Slinky5890** [comment]
holy


shit

File: code/modules/admin/tickets/admin_ticket.dm [hunk 1]
```diff
@@ -40,7 +41,11 @@ var/ticket_counter_visible_to_everyone = 0
 /datum/ticket_log/proc/toLogString()
 	return "[isAdminComment() ? "COMMENT - " : ""][istype(user, /client) ? key_name_params(user, 0, 1, null, parent) : user] - [text]"
 
+/datum/ticket_log/proc/fix_apostrophes()
```
Comment (AshCorr) [inline code/modules/admin/tickets/admin_ticket.dm]: You might want to move this code to a proc that can be used everywhere.

If you put the code in https://github.com/yogstation13/yogstation/blob/master/code/__HELPERS/text.dm and make it take a string as a parameter you can then return the santizied string.

**Review by AshCorr — CHANGES_REQUESTED** [review]

**Review by AshCorr — APPROVED** [review]

**Pull request merged**